config.prod.yaml

```diff
@@ -10,11 +10,13 @@ worker_heap_limit_mb: 500
 # Logger info
```

Comment:
Should config.dev.yaml be updated too? to make sure people dont think using gelf stream is supported at all

Reply:
Good idea!

Updated PR.